release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -165,6 +165,35 @@ For more information, see xref:../installing/installing_aws/installing-restricte
 [id="ocp-4-11-web-console"]
 === Web console
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic plug-in updates
+
+With this update, you can now enable and disable plug-ins in the *Administration* -> *Cluster Settings* page of the web console. You can also add different extension types to customize your plug-in.
+
+For more information, see xref:../web_console/dynamic-plug-ins.adoc#dynamic-plug-ins_web_console[Dynamic plug-ins].
+
+[id="ocp-4-11-dark-mode"]
+==== Support for dark mode theme
+
+The {product-title} web console now supports the dark mode theme. On the *User Preferences* page, select your preferred mode to view the web console in.
+
+[id="ocp-globally-installed-operators-display-operand-instances"]
+==== Display operand instances for all managed namespaces on the *Installed Operator* page
+
+With this update, the default view for globally installed Operators in the *All namespaces* and *Operand* tabs of the *Installed Operator* -> *Details* page will show all Operators across all namespaces. You are still able to view only the instances in the selected namespace within the project selector. When viewing the operand instances, a new radio switching control allows all operand instances from either all namespaces or only the current namespace to be seen.
```

Comment:
@XiyunZhao PTAL. Thank you!

Reply:
`the default view for globally installed Operators in the *All namespaces* and *Operand* tabs of the *Installed Operator* -> *Details* page will show all Operators across all namespaces`
@opayne1 The operator details page has not been updated, it is recommended to remove this sentence to avoid misleading. Other parts, it;s ok for me

Reply:
@XiyunZhao I have updated to reflect your comments. Thank you!